fakeredis/aioredis.py

```diff
@@ -1,5 +1,15 @@
-try:
-    import aioredis
+import redis
+import packaging.version
+
+# aioredis was integrated into redis in version 4.2.0 as redis.asyncio
+if packaging.version.Version(redis.__version__) >= packaging.version.Version("4.2.0"):
+    import redis.asyncio as aioredis
+    from ._aioredis2 import FakeConnection, FakeRedis  # noqa: F401
+else:
+    try:
+        import aioredis
+    except ImportError as e:
+        raise ImportError("aioredis is required for redis-py below 4.2.0") from e
     import packaging.version
```

Comment:
No need for this import as it is imported on top

Reply:
Ah, my bad! My editor wasn't picking this up automatically :grimacing: